mantidproject/mantid
# PhaseQuad integration

**Description of work.**
The phasequad part of the muon GUI was not fully integrated. It was not obvious how to add it to the analysis, particularly in time domain.



**To test MA:**
1. Open muon analysis
2. Load MUSR 62260
3. Go to the phase tab
4. Try to create a ohasequad -> get a warning
5. Make a phase table, then make a phasequad
6. You should now have 2 new workspaces in the ADS (Re and Im parts)
7. Check that the new phasequads show up on the plots
8. Go to grouping add a pair
9. Unselect all of the groups
10. Go to fitting and add a fitting function
11. Do a fit
12. Go to sequential fit and do a series of fits
13. Load some new runs (62261-62263)
14. The phasequad's will have vanished from the plots
15. Do a phasequad
16. Check you have 4 new workspace (Re and Im for both 62261 and 62263)
17. Close muon analysis

**To test FDA:**
1. Open frequency domain analysis
2. Load MUSR 62265
3. Go to transform
4. Do an FFT on the phasequad data
5. Go to fitting and do a fit

Fixes #30061 . 






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
